quickwit/quickwit-common/src/uri.rs

```diff
@@ -38,6 +38,7 @@ pub enum Protocol {
     PostgreSQL,
     Ram,
     S3,
+    Http,
```

Comment:
```suggestion
    Http { secure: bool },
```

Reply:
Shall we handle `https` as well?

Reply:
I thought also about using `Grpc` protocol. For example, s3 is using HTTP requests so http seems a bit misleading here.